metriq_gym/quantinuum/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""Quantinuum provider integration via qNexus.
```

Comment:
nice, we should have a header like this also for the `__init__.py` file of the `/local` folder